src/components/util/document-store.tsx

```diff
@@ -0,0 +1,28 @@
+import { getWalletDetails } from "./wallet";
+
+export const setDocumentStoreInformation = async (documentStoreAddress: string): Promise<void> => {
+  const walletDetails = await getWalletDetails();
+  if (walletDetails) {
+    const storedInformation = localStorage.getItem(`
+    ${walletDetails.network}-${walletDetails.address.toLowerCase()}`);
+
+    let documentStoreArray: string[] = [];
+    if (storedInformation != null) {
+      documentStoreArray = JSON.parse(storedInformation);
+    }
+    documentStoreArray.push(documentStoreAddress);
+    localStorage.setItem(
+      `${walletDetails.network}-${walletDetails.address.toLowerCase()}`,
+      JSON.stringify(documentStoreArray)
+    );
+  }
+};
+
+export const retrieveDocumentStoreInformation = async (): Promise<Array<string>> => {
```

Comment:
overall same feedbacks as above

Reply:
okay changed to "retrieveDocumentStoreInLocalStorage"

Reply:
i think it works for both? as long as the developers/maintainer understand the context  :x